test/distributed/_shard/sharding_plan/test_sharding_plan.py

```diff
@@ -61,200 +56,6 @@ def build_plan(self, module: nn.Module) -> ShardingPlan:
 
 
 class TestShardingPlan(ShardedTensorTestBase):
-
-    @with_comms(init_rpc=False)
-    @skip_if_lt_x_gpu(TEST_GPU_NUM)
-    @requires_nccl()
-    def test_sharding_plan_simple_megatron(self):
```

Comment:
These tests are relied on Sharded Linear and we should also remove these two tests.